parlai/mturk/core/mturk_data_handler.py

```diff
@@ -104,6 +111,28 @@ def __init__(self, task_group_id=None, file_name='pmt_data.db'):
         self.table_access_condition = threading.Condition()
         self.create_default_tables()
 
+    @staticmethod
+    def save_world_data(prepped_save_data, task_group_id,
```

Comment:
just for my understanding, is this the structure of the directory:
run_data/
    sandbox/
    live/
        task_id(of this run?)/
            conversation_id/
                custom_data.json
                worker{id}.json

Reply:
- run_data
  |- sandbox (interior structure same as live)
  |- live
      |- {task_id}
          |- {conversation_id}
               |- custom
                   |- data.json
                   |- data.pickle
               |- workers
                   |- {worker_id}.json

Reply:
Just realized you had it essentially correct but github dropped out the spaces.